src/components/Reactions/ReactionsList.js

```diff
@@ -1,102 +1,79 @@
-/* eslint-disable */
+// @ts-check
 import React from 'react';
 import PropTypes from 'prop-types';
+// @ts-ignore
 import { NimbleEmoji } from 'emoji-mart';
 
 import { defaultMinimalEmojis, emojiSetDef, emojiData } from '../../utils';
 
-class ReactionsList extends React.Component {
-  static propTypes = {
-    /** List of reactions */
-    reactions: PropTypes.array,
-    /** Provide a list of reaction options [{id: 'angry', emoji: 'angry'}] */
-    reactionOptions: PropTypes.array,
-    /** If true, reaction list will be shown at trailing end of message bubble. */
-    reverse: PropTypes.bool,
-    /** Object/map of reaction id/type (e.g. 'like' | 'love' | 'haha' | 'wow' | 'sad' | 'angry') vs count */
-    reaction_counts: PropTypes.object,
-  };
-
-  static defaultProps = {
-    reactionOptions: defaultMinimalEmojis,
-    emojiSetDef,
-    reverse: false,
-    reactions: [],
-  };
+/** @type {React.FC<import("types").ReactionsListProps>} */
+const ReactionsList = ({
+  reactions,
+  reaction_counts,
+  reactionOptions = defaultMinimalEmojis,
+  reverse = false,
```

Comment:
Same question - https://github.com/GetStream/stream-chat-react/pull/345#discussion_r442169570

Reply:
just checked, and it picks them up alright.